<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapForm</class>
 <widget class="QWidget" name="mapForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Карта</string>
  </property>
  <widget class="QLabel" name="mapLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../q.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
